src/helpers.php

```diff
@@ -13,15 +13,16 @@
     function current_route(string $locale = null): string
     {
         $route = Route::getCurrentRoute();
-        $query = request()->getQueryString();
+        $parameters = request()->route()->parameters;
```

Comment:
We can use the `$route` variable from line 15 here. So it could be `$route->parameters;` instead.